src/XRayUS/XRayUSJSON.ts

```diff
@@ -135,6 +138,46 @@ namespace XRayUSJSON {
         NotRated: number;
     }
 
+    interface RiskStatistics {
+        AsOfDate: string;
+        TrailingTimePeriod: TrailingTimePeriod;
+        DataFrequency: 'Monthly' | 'Quarterly';
+        Portfolio: RiskStatisticsPortfolio;
+        Benchmark: RiskStatisticsBenchmark;
+        Security: Array<RiskStatisticsSecurity>;
+    }
+
+    type TrailingTimePeriod = (
+        | 'Year1'
+        | 'Year2'
+        | 'Year3'
+        | 'Year5'
+        | 'Year10'
+    );
+
+    interface RiskStatisticsItem {
+        Mean: number;
+        SharpeRatio: number;
+        StandardDeviation: number;
+    }
+
+    interface RiskStatisticsPortfolio extends RiskStatisticsItem {
+        SortinoRatio: number;
+        ExcessReturn: number;
+        InformationRatio: number;
+        TrackingError: number;
+    }
+
+    interface RiskStatisticsSecurity {
+        RiskStatisticsItem: RiskStatisticsItem;
+        SecurityId: string;
+        Weight: number;
+    }
+
+    interface RiskStatisticsBenchmark extends RiskStatisticsItem {
```

Comment:
According to docs, Benchmark and Portfolio have the same format:

<img width="821" height="413" alt="Image" src="https://github.com/user-attachments/assets/d06ab38e-baf8-4f3c-bb3f-c47c36ce522b" />